src/main/resources/openapi/schemas/ingestJobRecordRequest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "description": "Ingest job record request",
+  "type": "object",
+  "properties": {
+    "localId": {
+      "type": "string",
+      "description" : "Local identifier"
+    },
+    "payload": {
```

Comment:
I think this is OK for now, but given the latest input we will need to add two different properties to hold the "marcPayload" and "inventoryPayload". Both likely structured properties (JSON)

Reply:
ok , so object.

Reply:
And we still use the term marc in 2022? It would never be anything?